src/ess.jl

```diff
@@ -480,7 +482,7 @@ function _expectand_proxy(::typeof(StatsBase.mad), x)
     return _expectand_proxy(Statistics.median, x_folded)
 end
 function _expectand_proxy(f::Base.Fix2{typeof(Statistics.quantile),<:Real}, x)
-    y = similar(x, Bool)
+    y = similar(x)
```

Comment:
Necessary because otherwise our transformation discards the type of `x`, so a `Float32` eltype `x` will get a `Float64` ESS.